README.md

```diff
@@ -57,6 +58,19 @@ dependencies {
     compileOnly 'com.destroystokyo.paper:paper-api:1.16.4-R0.1-SNAPSHOT'
 }
 ```
+> Kotlin DSL
+ * Repository:
+```kotlin
+repositories {
+    maven(url = "https://papermc.io/repo/repository/maven-public/")
```

Comment:
is this url param name required here? seems unnecessary to me, but am not at a pc rn

Reply:
I agree the `url = ` is unnecessary. I tend to exclude it unless I am also providing the name param.

Reply:
Yeah, needs to be removed.